nvd3/templates/content.html

```diff
@@ -96,6 +96,12 @@
 
     {% endblock custoattr %}
 
+    {% block y_axis_scale %}
+    {% if chart.y_axis_scale_min or chart.y_axis_scale_max %}
```

Comment:
It adds the possibility to set either min or max. So you can set `min=-1` but leave it 'open' towards the maximum by just not specifying it.